### ISSUE TITLE
Cannot put debug points to query in or out inside a partition

### ISSUE THREAD

**Issue by RAVEENSR** [wso2/product-sp#95]
**Description:**
Cannot put debug points to query in or out inside a partition

**Affected Product Version:**
WSO2 Stream Processor 4.0.0 Milestone 13 Release

**OS, DB, other environment details and versions:**    
Ubuntu 16.04

**Steps to reproduce:** 
try to add debug points to follwing siddhi app.
-------------SIddhi app------>
@app:name('TestSiddhiApp')
define stream sensorStream (sensorId string, temperature double);
partition with (sensorId of sensorStream)
begin
   from sensorStream#window.length(1)
   select max(temperature) as maxTemp
   insert into DeviceTempStream
end;
----------End of siddhi app---->

**Comment by RAVEENSR** [comment]
This issue is not fixed yet. (The commit referred to here is a mistake)

**Comment by this** [comment]
Seems like this a limitation in Siddhi. @suhothayan Please confirm.

**Comment by suhothayan** [comment]
Yes its a limitation in Siddhi we'll be fixing this in the future
